tests/execution_engine/test_redshift_execution_engine.py

```diff
@@ -0,0 +1,80 @@
+import pytest
+from sqlalchemy.sql.sqltypes import BOOLEAN, CHAR, INTEGER, VARCHAR
+
+from great_expectations.execution_engine.redshift_execution_engine import RedshiftExecutionEngine
+from great_expectations.validator.computed_metric import MetricValue
+from great_expectations.validator.metric_configuration import (
+    MetricConfiguration,
+    MetricConfigurationID,
+)
+
+
+@pytest.mark.unit
+def test_redshift_execution_engine_resolve_metrics(mocker):
+    # arrange
+    engine = RedshiftExecutionEngine(
```

Comment:
I would call this `execution_engine` because I think you want to mock the `execution_engine.engine` [here](https://github.com/great-expectations/great_expectations/blob/711901a65cad4f0b06ccef284d6d1b608ea14a40/great_expectations/execution_engine/sqlalchemy_execution_engine.py#L274) (SQL Alchemy engine).